extensions/cpsection/modemconfiguration/model.py

```diff
@@ -319,7 +319,7 @@ def _get_initial_config(self):
         if not provider_name:
             provider_name = ''
         else:
-            provider_name = provider_name.decode('utf-8')
+            provider_name = provider_name
```

Comment:
I would suggest you to remove the `else` block as it doesn't make sense (if the change you are intending is right)

Reply:
You are right. Fixed in cfec35e

Reply:
:+1: